resource_calendar_rrule/static/src/xml/resource_calendar_rrule.xml

```diff
@@ -12,11 +12,6 @@
         <t t-if="value.type == 'odd'">
             <t t-set="data_keys" t-value="['data', 'data_odd']" />
         </t>
-        <t t-if="value.type == undefined">
```

Comment:
also with your changes, we can end up here. A forced other default, a value just happening to be null, that kind of stuff. I think what we need here is what the TODO says, a little link/button that makes a call to create some default calendar.

Reply:
Just wish to be a bit more convinced that that is the better solution... currently when value.type is `undefined` we will show a calendar filled with zeros, and people can fill them in as they please. In what situation can that go wrong? If it's only about a link/button, that can also be achieved by having it execute a Javascript function that unhides the input elements?

Reply:
think about what happens if some code writes a null value in the field

Reply:
I just wrote an empty dict to the `simplified_attendance` field. `record.read(['simplified_attendance'])` now gives me `[{'id': 90, 'simplified_attendance': {}}]`. The form view for the calendar shows me:

![image](https://user-images.githubusercontent.com/1466356/34321408-cfbf5f86-e80e-11e7-8cef-1a1254c98cb6.png)

Reply:
yes, and that's wrong. We need to be able to make a difference between: "Doesn't work at all for this contract" (zero hours contract) and "No schedule set [yet]" (null)

What you propose here is basically the in my opinion very bad Idea of float and integer fields that null is 0. Bad.

Reply:
Alright, agreed, but that also means we should scrap [this default](https://github.com/OCA/hr/blob/8.0/resource_calendar_rrule/models/resource_calendar.py#L23) and replace it by the null-equivalent?

Reply:
not so sure actually. Defaults are nice. It's just that for other modules using us, they should be able to change it if they want. Or an admin can override this with some configuration. And then handling null values the way people should expect is a good thing :-)

What I notice is that a possibility to clear out a calendar then should be added too some time, but as I consider it bad style to ask for features during review, I wont

Reply:
Have implemented the null state, and the button to return to it after user had filled something.

Reply:
nice! now we just need to get travis green